lib/shopify_transporter/exporters/magento/soap.rb

```diff
@@ -33,10 +33,14 @@ def call_in_batches(method:, params: {}, batch_index_column:)
 
               $stderr.puts "Processing batch: #{current_id}..#{end_of_range}"
 
-              enumerator << call(
-                method,
-                params.merge(batching_filter(current_id, end_of_range, batch_index_column)),
-              )
+              begin
+                enumerator << call(
+                  method,
+                  params.merge(batching_filter(current_id, end_of_range, batch_index_column)),
+                )
+              rescue Savon::Error
+                $stderr.puts "Skipping batch: #{current_id}..#{end_of_range} after #{MAX_RETRIES} retries because of an error."
```

Comment:
Do you mind adding the error details here as well?
```
 $stderr.puts 'The exact error was:'
... e.class + e.message
```

Reply:
Sure!